app/src/main/java/org/fossasia/openevent/general/event/EventUtils.kt

```diff
@@ -16,10 +16,12 @@ object EventUtils {
 
     private val timeFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("hh:mm a")
     private val dateFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy")
+    private val frontendUrl: String = "https://open-event-frontend-dev.herokuapp.com/e/"
```

Comment:
This is dev environment. Not necessary the event will be there

Reply:
Actually in today's meeting we were asked to change it to this one. I also tested it manually for some events and the links were always working. What do you suggest should be done ?

Reply:
Then it's fine